### PR TITLE
gui homework

### PR DESCRIPTION
**Pull request by NMandel527**

### CODE REVIEW

**Comment by NMandel527** [comment]
GUI Projectile

File: src/main/java/mandel/projectile/ProjectileFrame.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                Projectile projectile = new Projectile(angleSlider.getValue(),
+                        Double.parseDouble(velocityField.getText()));
+                projectile.setSeconds(Double.parseDouble(secondsField.getText()));
+
+                x.setText(Double.toString(projectile.getX()));
+                y.setText(Double.toString(projectile.getY()));
+                peakY.setText(Double.toString(projectile.getPeakY()));
+                interceptX.setText(Double.toString(projectile.getInterceptX()));
```
Comment (andrewoid) [inline src/main/java/mandel/projectile/ProjectileFrame.java:73]: Get rid of the duplicate code.

Comment (NMandel527) [reply src/main/java/mandel/projectile/ProjectileFrame.java:73]: I tried to make it non-repetitive but it wasn't working well. Is the best way to do it creating a new method and passing everything into it and then calling it each time?

File: src/main/java/mandel/projectile/ProjectileFrame.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                calculations(angleSlider, velocityField, secondsField, x, y, peakY, interceptX);
+            }
+        };
+
+        velocityField.getDocument().addDocumentListener(docListener);
+        secondsField.getDocument().addDocumentListener(docListener);
+    }
+
+    public void calculations(JSlider angleSlider, JTextField velocityField, JTextField secondsField,
+                              JLabel x, JLabel y, JLabel peakY, JLabel interceptX) {
```
Comment (andrewoid) [inline src/main/java/mandel/projectile/ProjectileFrame.java:88]: If you make these into member variables then they done need to be passed in

**Review by andrewoid — APPROVED** [review]

**Pull request merged**